mtgjson5/arg_parser.py

```diff
@@ -22,51 +17,57 @@ def parse_args() -> argparse.Namespace:
     """
     parser = argparse.ArgumentParser("mtgjson5")
 
+    parser.add_argument(
+        "--use-envvars",
+        action="store_true",
+        help="Use environment variables over parser flags for build operations",
+    )
+
     # What set(s) to build
     sets_group = parser.add_mutually_exclusive_group()
     sets_group.add_argument(
-        "-s",
         "--sets",
+        "-s",
```

Comment:
why?

Reply:
By reordering, I changed how the help menu displays to show the full command instead of the shorthand. This is a non-breaking change and aims to upgrade how we help our users